src/Compiler/AbstractIL/ilreflect.fs

```diff
@@ -1948,12 +1855,6 @@ let rec buildMethodPass2 cenv tref (typB: TypeBuilder) emEnv (mdef: ILMethodDef)
         // p.CharBestFit
         // p.NoMangle
 
-#if !FX_RESHAPED_REFEMIT || NETCOREAPP3_1
```

Comment:
Just a sanity check: the first condition is now always false, but can't the second one still be true for some case? That would mean we'd still need that code I guess.

Reply:
We now always invoke via reflection, netstandard2.0 doesn't have the api defined, so we used reflection to invoke it.  So the api existed as a public api in the desktop and from netcoreapp3.1, and existed as a private Api for netstandard2.0 and I think netstandard2.1.

Now instead of invoking it directly in all cases we use reflection, it saves us having two different dll's and costs us a small bit of reflection.

We are required to target netstandard 2.0 rather than netcoreapp for the .dll because the compiler service can target the windows desktop framework.  For sure that is necessary for Us because Visual Studio still runs on the desktop framework.

Reply:
Thanks for the explanation!